### PR TITLE
New website design.

### PR DESCRIPTION
**Pull request by ggbecker**
Use a different theme: https://github.com/artemsheludko/flexible-jekyll

### Front Page
preview_blog1

--------------------

### Post page

preview_blog2

### Updates

- removed thumbnails
- updated textbox highlight to match old colors and format from previous blog design.

### CODE REVIEW

**Comment by Mab879** [comment]
Do we know how well this theme will work with accessability technologies?

**Comment by ggbecker** [comment]
> Do we know how well this theme will work with accessability technologies?

I mean, I can tab between links and access the posts. But we should ask Vojta for more details.

**Comment by ggbecker** [comment]
Vojta has tried the new design and resulted on a small improvement: [bfa4742](https://github.com/ComplianceAsCode/ComplianceAsCode.github.io/pull/39/commits/bfa4742a868751d38fd57621978383061c27ef57)

**Comment by Mab879** [comment]
Please take a look at release_0_1_54.png it seems to have the release number on it twice.

**Comment by ggbecker** [comment]
> Please take a look at release_0_1_54.png it seems to have the release number on it twice.

Yeah, I notice that. Fixed in f0a6a1c8ded4c908cdf160da89c4ae46e61a2f66

**Review by Mab879 — COMMENTED** [review]
If we are not using the thumbnails should we remove them?

File: 404.html [hunk 1]
```diff
@@ -1,24 +0,0 @@
----
```
[the remark this answers is not in the document]
Comment (ggbecker) [reply 404.html:1]: that's true. I'll take a look.

**Comment by ggbecker** [comment]
> If we are not using the thumbnails should we remove them?

we probably can. I just didn't want to delete them right away, maybe we can still use them at some point.

**Comment by matejak** [comment]
I really like the new look.
The only question that comes up would be - would it be possible to include the author name on the front page? If so, I would suggest to add it, but if not, then it doesn't mind, the new style looks awesome.

**Comment by ggbecker** [comment]
> I really like the new look. The only question that comes up would be - would it be possible to include the author name on the front page? If so, I would suggest to add it, but if not, then it doesn't mind, the new style looks awesome.

addressed in https://github.com/ComplianceAsCode/ComplianceAsCode.github.io/pull/39/commits/d86a33c64ccdc141db4a6f5f8923821367ad143f

**Pull request merged**